package.json

```diff
@@ -5,7 +5,9 @@
   "main": "bot.js",
   "dependencies": {
     "curl": "^0.1.4",
-    "irc": "^0.5.0"
+    "irc": "^0.5.0",
+    "mathjs": "^13.0.1",
+    "sox": "^0.1.0"
```

Comment:
Unless we actually start using sox inside of node we don't need this module. Right now we call it externally. This could be another future refactor though I'm not sure if its worth the time.